tests/integration/Runner.php

```diff
@@ -81,6 +81,10 @@ private function getRunnableServices($service, $version)
         return $services;
     }
 
+    private function toCamelCase($word, $separator = '_') {
```

Comment:
This does not adhere to [PSR-2 coding style](http://www.php-fig.org/psr/psr-2/). Would you mind taking a quick look at the standard for future reference? :smile: It should be:

```php
private function toCamelCase($word, $separator = '_')
{
    return str_replace($separator, '', ucwords($word, $separator));
}
```